oak_functions/loader/src/server.rs

```diff
@@ -238,6 +249,21 @@ impl WasmState {
         })
     }
 
+    // Writes the given `buffer` by allocating `buffer.len()` Wasm memory and writing the address of
```

Comment:
Use doc comment with `///` instead of `//` :)

Reply:
Done.